_includes/tags.html

```diff
@@ -17,13 +17,21 @@
                        tags are used on a blog article then path should be
                        '/blog/'. Remember to leverage vars.path instead of
                        using the literal string '/blog/'. Path is used to
-                       create the filtered URL: {{ path }}?filter_tags={{ tag }}
+                       create the filtered URL: {{ path }}?filter_tags={{ tag }}.
+
+   display_header:     The optional boolean flag to display the tags_header element.
+                       Defaults to True. Adds class 'tags__hide-header' if falsy
+                       value passed (e.g., '', False, None).
+
+   display_inline:     The optional boolean flag to display the tags list inline.
+                       Defaults to True. Adds class 'tags__block-list' if falsy
+                       value passed.
 
    ========================================================================== #}
 
-{% macro render(tags, path) %}
-<div class="tags">
-    <h1 class="tags_header">Topics:</h1>
+{% macro render(tags, path, display_header=True, display_inline=True) %}
+<div class="tags {{'tags__hide-header' if not display_header}} {{'tags__block-list' if not display_inline}}">
```

Comment:
Should this logic be simplified with `hide_header` and `display_block` rather than `not` conditions?